app/inputs/attachment_preview_input.rb

```diff
@@ -0,0 +1,9 @@
+class AttachmentPreviewInput < SimpleForm::Inputs::FileInput
+  def input(wrapper_options = nil)
+    out = ''
+    if object.send("#{attribute_name}?")
+      out << template.link_to(I18n.t('helpers.download'), object.send(attribute_name).url, class: 'btn btn-success btn-xs')
```

Comment:
Line is too long. [123/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.